web/app/js/components/ResourceDetail.jsx

```diff
@@ -227,13 +245,15 @@ export class ResourceDetailBase extends React.Component {
           upstreamMetrics,
           downstreamMetrics,
           edges,
+          resourceDefinition, // eslint-disable-line react/no-unused-state
           lastMetricReceivedTime,
           isTcpOnly,
           isTrafficSplit,
           loaded: true,
           pendingRequests: false,
           error: null,
-          unmeshedSources: this.unmeshedSources // in place of debouncing, just update this when we update the rest of the state
+          unmeshedSources: this.unmeshedSources, // in place of debouncing, just update this when we update the rest of the state
+          queryForDefinition: true,
```

Comment:
the value of `queryForDefinition` shouldn't have changed here from the initial state, should it? do we need this line?

Reply:
Not, I'll remove the line